Code/GraphMol/MolStandardize/AcidBaseCatalog/acid_base_pairs.in

```diff
@@ -0,0 +1,56 @@
+//
+//  Copyright (C) 2021 Greg Landrum
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+namespace defaults {
+// The default list of AcidBasePairs, sorted from strongest to weakest.
+// This list is derived from the Food and Drug: Administration Substance
+// Registration System Standard Operating Procedure guide.
+//
+//	Name	Acid	Base
+const std::vector<std::tuple<std::string, std::string, std::string>>
```

Comment:
The one drawback to this technique is this vector+string will always get initialized at load time even if the AcidBase pairs aren't used.

I doubt it is a problem here, but if it is we can always make an init once function that creates (only once) and returns this on demand.